pkg/registry/apps/deployment/storage/storage.go

```diff
@@ -58,7 +58,7 @@ type DeploymentStorage struct {
 }
 
 // maps a group version to the replicas path in a deployment object
-var replicasPathInDeployment = fieldmanager.ResourcePathMappings{
+var ReplicasPathInDeployment = fieldmanager.ResourcePathMappings{
```

Comment:
Making it public is fine, but it should be returned by a function so that people can't modify it. And let's move the test to integration, thanks!

Reply:
Makes sense, will amend the commit